### PR TITLE
Added bib2md script

### PR DESCRIPTION
**Pull request by JoschD**

### CODE REVIEW

File: scripts/bib2md.py [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+    print_markdown(collection, citekeys, add_year)
+
+
+def read_bibfile(bibfile: Path) -> dict:
+    """ Reads the bib-file and converts it's content to a dictionary.
+    The entries are sorted by citekey and additional two entries
+    `entrytype` (e.g. article, report etc.) and `rawtext` (the bib-raw text)
+     are created.
+    """
+    with open(bibfile, 'r') as f:
```
Comment (fsoubelet) [inline scripts/bib2md.py:59]: not `bibfile.read_text()`?

File: scripts/bib2md.py [hunk 2]
```diff
@@ -0,0 +1,209 @@
+"""
+Prints markdown for the Publications.md from .bib files.
+
+Usage: Call `main` with the path to the script and a list of citekeys.
```
Comment (fsoubelet) [inline scripts/bib2md.py:4]: Maybe let's add a little bit of info on what `the script` is

File: scripts/bib2md.py (hunk 1 above)
Comment (JoschD) [reply scripts/bib2md.py:59]: No, I wanted it line by line

File: scripts/bib2md.py (hunk 2 above)
Comment (JoschD) [reply scripts/bib2md.py:4]: Well, "the script" is obviously a typo. It's supposed to read "the bibfile"

**Review by fsoubelet — APPROVED** [review]
👍

**Pull request merged**